### PR TITLE
Update requests library

### PR DESCRIPTION
**Pull request by cjwebb**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-requests==2.8.1
+requests>=2.20.0
```
Comment (colin-lamed) [inline requirements.txt:1]: should we pin the version of requests?

Comment (cjwebb) [reply requirements.txt:1]: `requests~=2.20` ?
https://www.python.org/dev/peps/pep-0440/#compatible-release

Or were you thinking an explicit version? #91 asks for a relaxation of that kind of thing.

Comment (colin-lamed) [reply requirements.txt:1]: They want to embed service-manager in other projects?
a compatible release version sounds like the best thing?

Comment (cjwebb) [reply requirements.txt]: I have updated the requirements.txt and setup.py to use `~2.20`.
Please can you re-approve?

**Pull request merged**